dev/io.openliberty.microprofile.metrics.internal.5.x.monitor_fat/fat/src/io/openliberty/microprofile/metrics/internal/monitor_fat/ComputedMetricsTest.java

```diff
@@ -53,8 +55,13 @@ public class ComputedMetricsTest {
     private static final String[] ALWAYS_EXPECTED_COMPUTED_METRICS = new String[]{
             "cpu_processCpuUtilization_percent{mp_scope=\"vendor\",}",
             "memory_heapUtilization_percent{mp_scope=\"vendor\",}",
-            "gc_time_per_cycle_seconds{mp_scope=\"vendor\",name=\"global\",}",
-            "gc_time_per_cycle_seconds{mp_scope=\"vendor\",name=\"scavenge\",}"
+            "gc_time_per_cycle_seconds{mp_scope=\"vendor\","
```

Comment:
why reduce?

Reply:
In the SOE builds, we run using different versions of the JDK, where each provides a different name for the gc, that we can only know during runtime.  Hence, the test reduces the string to find all the `gc_time_per_cycle_seconds` metrics and verify its correctness.